cmd/zed/dev/indexfile/lookup/command.go

```diff
@@ -30,18 +30,18 @@ of any nested records.`,
 }
 
 func init() {
-	zedindex.Cmd.Add(Lookup)
+	indexfile.Cmd.Add(Lookup)
 }
 
 type LookupCommand struct {
-	*zedindex.Command
+	*indexfile.Command
 	keys        string
 	outputFlags outputflags.Flags
 	closest     bool
 }
 
 func newLookupCommand(parent charm.Command, f *flag.FlagSet) (charm.Command, error) {
-	c := &LookupCommand{Command: parent.(*zedindex.Command)}
+	c := &LookupCommand{Command: parent.(*indexfile.Command)}
```

Comment:
I can't leave a comment at the site, but there's an error string down below that starts with "zed index lookup". I'd remove it, but if you keep it, change index to indexfile.